### PR TITLE
DOCSP-44963: Data modeling in find usage examples

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

*Note*: new structure was approved by the PM
JIRA -https://jira.mongodb.org/browse/DOCSP-44963
Staging - 
https://deploy-preview-137--docs-rust.netlify.app/usage-examples/findOne/
https://deploy-preview-137--docs-rust.netlify.app/usage-examples/find/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/usage-examples/find.txt [hunk 1]
```diff
@@ -26,54 +39,100 @@ Example
...
+
+         ...
+         Some(
+            Document({
+               "_id": ObjectId(
+                     "...",
+               ),
+
+               ...
+
```
Comment (rustagir) [inline source/usage-examples/find.txt:96]: S: can we shorten these

**Review by rustagir — APPROVED** [review]
lgtm with a formatting suggestion

File: source/usage-examples/find.txt (hunk 1 above)
Comment (norareidy) [reply source/usage-examples/find.txt:96]: restaurant_id is probably unnecessary, so I'll take that out

**Review by abr-egn — APPROVED** [review]
Very nice, this is a great way to contrast using the base `Document` type vs. the `serde` support 🙂

**Pull request merged**